windows/d2-installer.nsi

```diff
@@ -355,10 +367,13 @@ Function DetectVSAndSDK
     StrCpy $VCPath $0
     StrCpy $VCVer $1
 
-    ReadRegStr $0 HKLM "Software\Microsoft\Windows Kits\Installed Roots" "KitsRoot81"
+    ReadRegStr $0 HKLM "Software\Microsoft\Microsoft SDKs\Windows\v10.0" "InstallationFolder"
+    IfErrors 0 done_sdk
+    ClearErrors
+    ReadRegStr $0 HKLM "Software\Microsoft\Microsoft SDKs\Windows\v8.1" "InstallationFolder"
```

Comment:
I didn't remember why we used these `KitsRoot` entries. For me they are identical to the `Microsoft SDK` entries for the versions without an A. I think we should add a test for the existence of `kernel32.lib` in the expected lib folder.